CONTRIBUTING.md

```diff
@@ -306,9 +313,28 @@ $ ./cli exec --authn-oidc
 root@9feae5e5e001:/src/conjur-server#
 ```
 
+###### Run Cukes with Google Cloud Platform (GCP) Compatible Environment
+
+**Prerequisites**
+- A Google Cloud Platform account. https://cloud.google.com/.
+- Google Cloud SDK installed. https://cloud.google.com/sdk/docs
+- Access to a running Google Compute Engine instance. 
+
+To run the cukes with Google Cloud Platform (GCP) compatible environment, run `cli`
```

Comment:
with a Google

Reply:
?

Reply:
"To run the cukes with Google Cloud Platform"
Should be
"To run the cukes with a Google Cloud Platform"